src/applications/check-in/utils/appointment/index.js

```diff
@@ -165,6 +165,60 @@ const sortAppointmentsByStartTime = appointments => {
     : [];
 };
 
+/**
+ * @param {Array<Appointment>} appointments
+ */
+
+function organizeAppointmentsByYearMonthDay(appointments) {
+  const organizedData = [];
+  const months = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ];
+  const days = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
```

Comment:
these day abbreviations also need to be translatable

Reply:
Refactored and removed these